docs/exchanges/kucoin.md

```diff
@@ -8,16 +8,20 @@ tags:
 
 ## 📁 [Connector folder](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/exchange/kucoin)
 
+## 📝 Signup
+
+* Hummingbot referral link - <https://www.kucoin.com/ucenter/signup?rcode=272KvRf>
+* Sign up with Kucoin using Hummingbot's referral link for a 10% discount!
```

Comment:
Are you sure that Kucoin users get 10% off? Check with Luciana and refer to the KuCoin page https://www.notion.so/hummingbot-foundation/KuCoin-6668f089044a473499d7530522cfb142